Tests/ClientRuntimeTests/SerializationTests/SerializationUtilsTests/TimestampSerdeUtilsTests.swift

```diff
@@ -31,6 +31,8 @@ class TimestampSerdeUtilsTests: XCTestCase {
 
     // MARK: - Encoding Tests
 
+    // TODO: Fix this test on Linux
```

Comment:
This seems like a weird todo. Maybe at least we should investigate why this is failing and add the details.

Reply:
I don't want to spend more time on this right now, so I filed https://github.com/awslabs/aws-sdk-swift/issues/1006 so that these disabled tests are tracked & can be prioritized against other work.

Reply:
I left a comment by each disabled test with a link to the ticket as well.